app/models/address.rb

```diff
@@ -7,5 +7,5 @@ class Address < ApplicationRecord
   enum address_type: { permanent: 0, temporary: 1 }
 
   #validation
-  validation :street_1, :street_2, :city, :state, :pincode, presence: true
+  validates :street_1, :street_2, :city, :state, :pincode, presence: true
```

Comment:
@RaviMariya-BTC All the validation is only with presence, You should wrap the validation with required regex like for example Pincode should not more than 6 digits and it should always an integer.